fs_src/index.html

```diff
@@ -540,6 +540,13 @@ <h1 id="head">Window</h1>
           <label><span id="close_spinner"></span>Close</label>
         </button>
       </div>
+      <form class="form-control" id="set_tgt_pos_form">
+        <label>Set to Postion</label>
+        <input type="number" id="set_tgt_pos_input">
+        <button id="set_tgt_pos_button" type="submit">

```

Comment:
Remove the type=submit

Reply:
This way the enter key would not work that is why I choose to go for form + submit. Is this breaking anything?

Reply:
I see what you mean, but then this field will behave different then any other, so we should not add this feature to just one of them

Reply:
okay I see, I can adjust the other fields too, this will be straight forward.

Reply:
Maybe in a different PR

Reply:
okay I adjusted the code, please review again.

Reply:
You missed to remove the type attr

Reply:
good catch